### PR TITLE
Feat/ga/91 content tree redesign

### PR DESCRIPTION
**Pull request by unseen1980**
# Issue

https://github.com/SAP/guided-answers-extension/issues/91

## Description
Screenshot 2022-06-21 at 16 18 08

Screenshot 2022-06-22 at 11 11 38

## Checklist for Pull Requests

- [ ] Supplied as many details as possible on this change
- [ ] Included the link to the associated issue in the Issue section above
- [ ] The code conforms to the general [development guidelines](https://github.com/SAP/guided-answers-extension/blob/main/docs/developer-guide.md).
- [ ] The code has **unit tests** where applicable and is easily unit-testable
- [ ] This branch is appropriately named for the associated issue

### CODE REVIEW

File: .changeset/poor-bees-pull.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'@sap/guided-answers-extension-core': minor
```
Comment (Klaus-Keller) [inline .changeset/poor-bees-pull.md:2]: I don't see changes in `@sap/guided-answers-extension-core`, I think we can skip this
```suggestion
```

**Review by Klaus-Keller — CHANGES_REQUESTED** [review]

**Comment by Klaus-Keller** [comment]
@unseen1980, no clue why SonarCloud shows 
```
0.0% Coverage
```

I see you added tests for `GuidedAnswerNavPath` and if I run test report local, it is shown:




Somehow this is not picked up by SonarQube, @kranthie-sap, any idea?

File: .changeset/poor-bees-pull.md (hunk 1 above)
Comment (unseen1980) [reply .changeset/poor-bees-pull.md:2]: I don't know why is picking it up? I can remove it
Screenshot 2022-06-23 at 10 54 06

Comment (Klaus-Keller) [reply .changeset/poor-bees-pull.md:2]: No idea, perhaps you need to merge latest `main` branch? In file list of this PR nothing changed.

Comment (unseen1980) [reply .changeset/poor-bees-pull.md]: ok removed 👍

**Review by Klaus-Keller — APPROVED** [review]
Code looks good, tests exist, build and tested locally. 
Approved

**Comment by unseen1980** [comment]
Thanks!

**Pull request merged**